doc/aspice_swe3_4/example/iceoryx_component/include/example_component/example_component.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
What if my new company Bobert Rosch Inc. wants to contribute code to iceoryx? Should we add an example with a non Bosch contributor?

Reply:
Not necessary because we have in https://github.com/eclipse/iceoryx/blob/master/CONTRIBUTING.md#header a generic example ready. i linked to it from here.